src/components/react-hook-form/ExpandingTextField.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useState } from 'react';
+import { InputAdornment } from '@mui/material';
+import { useMemo, FunctionComponent } from 'react';
+import { useFormContext, useWatch } from 'react-hook-form';
+import { TextInput } from '../..';
+
+interface ExpandingTextFieldProps {
+    name: string;
+    maxCharactersNumber?: number;
+    rows?: number;
+    minRows?: number;
+    maxRows?: number;
+    sx?: any;
+    label?: string;
```

Comment:
```suggestion
    label?: string;
    {...TextFieldProps} : any;
```

Reply:
i finally figured it out, i made the interface extands TextInputProps